File: timeClient.py
#cerner_2tothe5th_2022
#cerner_2^5_2022
import socket
soc = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
soc.connect(("localhost",49152))
soc.send("Pacific/Honolulu\n".encode())
data = soc.recv(1024)
print(data.decode())
soc.close()

File: timeClient.pl
#cerner_2tothe5th_2022
#cerner_2^5_2022
use strict;
use warnings;
use IO::Socket;
my $socket = new IO::Socket::INET (PeerAddr => 'localhost',PeerPort => '49152',Proto => 'tcp');
die "Could not create socket: $!n" unless $socket;
print $socket "Pacific/Honolulu\n";
print <$socket>;
close($socket);
